FppTest/component/active/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ set(UT_SOURCE_FILES
   "${CMAKE_CURRENT_LIST_DIR}/../tests/EventTests.cpp"
```

Comment:
Suggest alphabetizing this list. In vi, you can run `:[start line],[end line]!sort`.